File: Java/src/main/java/com/gildedrose/items/NormalItem.java
package com.gildedrose.items;

import com.gildedrose.Item;

public class NormalItem extends GildedItem {
    public NormalItem(Item item) { super(item); }

    @Override
    public void updateQuality() {
        decreaseQuality(1);
        decreaseSellIn();
        if (item.sellIn < 0)
            decreaseQuality(1);
    }
}

File: php/src/Items/NormalItem.php
<?php

declare(strict_types=1);

namespace GildedRose\Items;

use GildedRose\Item;
use GildedRose\Items\Abstract\AbstractItem;
use GildedRose\Items\Interface\ItemInterface;

class NormalItem extends AbstractItem implements ItemInterface
{
    /**
     * @param Item $item
     *
     * @return void
     */
    public function updateQuality(Item $item): void
    {
        $this->decreaseQuality($item, $this->getQualityDecrease($item));
    }
}
